### DISCUSSION TITLE
Gitignore specific file if local repo already has that file

### DISCUSSION THREAD

**Discussion by justinodunn** [community/community#49241] [Actions] [answered]
### Select Topic Area

Product Feedback

### Body

How do I make it so if my local repository doesn't have a specific file from the Github repo, the specific file will get pulled from the Github repo when I make the latest pull. However, if my local repository already has that specific file, and I pull the latest files from Github, gitignore will ignore that specific file from the Github repo, because my local repo already has that file.

Thanks!

**Comment by jsoref** [comment, marked as answer]
You can't.

That isn't what `.gitignore` does or how it works.

You can use a symlink maze if you need to emulate a behavior approximating this, although whether it will work will depend on how the code handles symlinks.

You should read https://git-scm.com/docs/gitignore
https://git-scm.com/docs/gitignore

**Reply by justinodunn** [reply]
Ok thanks so much.
